runtime/src/lib.rs

```diff
@@ -40,6 +40,8 @@ mod storage;
 pub mod telemetry;
 mod utils;
 pub use utils::*;
+#[cfg(feature = "iouring")]
+mod iouring;
```

Comment:
I suspect people will be *very interested in using this impl in the future. We'll keep private for now, but may want to open it up later.